cmd/goal/clerk.go

```diff
@@ -151,7 +151,7 @@ var clerkCmd = &cobra.Command{
 	},
 }
 
-func waitForCommit(client libgoal.Client, txid string) error {
+func waitForCommit(client libgoal.Client, txid string, transactionLastValidRound uint64) error {
```

Comment:
maybe we want 0 as a default value?

Reply:
Using the value of 0 would be useful for cases where we don't know what the last valid of a transaction. I don't believe we have such a use case, do we ? ( maybe if we refactor the `rawsendCmd` ).